### ISSUE TITLE
Trouble with CSS, JS and Delay

### ISSUE THREAD

**Issue by kerray** [mixu/electroshot#22]
Hi,
I think there's a bug - CSS and JS parameters stop doing anything once Delay and Height are specified. 

It doesn't really matter what url I try to screenshot.

These simple calls should demostrate it - ono the first one page is empty, on the second it's still there

electroshot https://www.seznam.cz 1024 --debug --css="body{display:none;}"

electroshot https://www.seznam.cz 1024x800 --delay 5000 --debug --css="body{display:none;}"

shot v1.4.0
electron v1.6.15
chrome v56.0.2924.87

**Comment by kerray** [comment]
OK, I just noticed the comment in target-window.js // Note that zoom-factor, selector, js and css are incompatible...

Nevermind then. How big is the chance this could be worked around?

**Comment by mixu** [comment]
The core limitation is that (at least when I wrote electroshot) the
preloaded.js script was only run after the page loaded which means any
stuff done via preloaded.js can only happen after a full page loaded which
may take an arbitrary number of milliseconds. Electron may have added a new
API or you might be able to do some kind of script injecting proxy to get
preloaded to run earlier. For my use cases the limitation wasn't a big deal
so I opted not to try to figure it out but happy to accept a PR!
